Userland/Applications/Browser/Tab.cpp

```diff
@@ -95,6 +97,7 @@ void Tab::view_dom_tree(const String& dom_tree)
 
 Tab::Tab(BrowserWindow& window, Type type)
     : m_type(type)
+    , m_filesystem_client(FileSystemAccessClient::construct())
```

Comment:
Let's construct the `FileSystemAccessClient` lazily as soon as it's needed for the first time, since most browser tabs will never initiate a download in their lifetime.